repo/content/index_blob_manager_test.go

```diff
@@ -184,14 +189,17 @@ func TestIndexBlobManagerStress(t *testing.T) {
 
 	for actorID := 0; actorID < numActors; actorID++ {
 		actorID := actorID
+		loggedSt := logging.NewWrapper(st, func(m string, args ...interface{}) {
+			t.Logf(m, args...)
+		}, fmt.Sprintf("actor[%v]:", actorID))
+		contentPrefix := fmt.Sprintf("a%v", actorID)
 
 		eg.Go(func() error {
-			loggedSt := logging.NewWrapper(st, t.Logf, fmt.Sprintf("actor[%v]:", actorID))
-
-			contentPrefix := fmt.Sprintf("a%v", actorID)
 			numWritten := 0
 			deletedContents := map[string]bool{}
-			ctx := testlogging.ContextWithLevelAndPrefix(t, testlogging.LevelDebug, fmt.Sprintf("actor[%v]:", actorID))
+			ctx := testlogging.ContextWithLevelAndPrefixFunc(t, testlogging.LevelDebug, func() string {
+				return fmt.Sprintf("actor[%v]@%v:", actorID, fakeTimeFunc().Format("150405.000"))
```

Comment:
Is logging going to cause time to advance?

Reply:
yes, but it's not really a goal to advance the time precisely here.